packages/documentation-framework/components/cssVariables/cssVariables.js

```diff
@@ -110,10 +100,9 @@ export class CSSVariables extends React.Component {
                   key={`${rowKey}_2`}
                   className="pf-v6-l-flex pf-m-column pf-m-align-self-center"
                 >
-                  <span
-                    className="ws-color-box"
-                    style={{ backgroundColor: value }}
-                  />
+                  <Icon>
+                    <CircleIcon color={value}/>
```

Comment:
Looks like the previous circle had a border and box shadow, which we probably want to retain at least one of those for when the circle's color is the same or really close to the background color.

And this is probably something for later, but we're showing the hex value for the color both as the text in the table (`#0066cc`) but also as the color of the icon (`color="#0066cc"`), which isn't accurate in dark theme. Technically this applies to other global vars, too, but mostly colors. For colors, you could just use the component var the table row is referencing for the circle color since component vars are available at the `:root` document scope. 

<img width="991" alt="Screenshot 2024-06-27 at 8 57 51 PM" src="https://github.com/patternfly/patternfly-org/assets/35148959/8b1345e4-c840-4676-96f6-9ea4d8cfe644">

<img width="971" alt="Screenshot 2024-06-27 at 8 58 10 PM" src="https://github.com/patternfly/patternfly-org/assets/35148959/e8a688a0-1138-4769-b940-e4316d56f442">